manageiq-operator/pkg/helpers/miq-components/postgresql.go

```diff
@@ -40,80 +44,94 @@ func postgresqlSecretName(cr *miqv1alpha1.ManageIQ) string {
 	return secretName
 }
 
-func NewPostgresqlConfigsConfigMap(cr *miqv1alpha1.ManageIQ) *corev1.ConfigMap {
+func NewPostgresqlConfigsConfigMap(cr *miqv1alpha1.ManageIQ) (*corev1.ConfigMap, controllerutil.MutateFn) {
 	labels := map[string]string{
 		"app": cr.Spec.AppName,
 	}
-	return &corev1.ConfigMap{
+
+	data := map[string]string{
+		"01_miq_overrides.conf": postgresqlOverrideConf(),
+	}
+
+	configMap := &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "postgresql-configs",
+			Name:      "miq-pg-configs",
 			Namespace: cr.ObjectMeta.Namespace,
-			Labels:    labels,
-		},
-		Data: map[string]string{
-			"01_miq_overrides.conf": postgresqlOverrideConf(),
 		},
 	}
+
+	mutateFn := configMapMutateFn(configMap, labels, data)
+
+	return configMap, mutateFn
 }
 
-func NewPostgresqlPVC(cr *miqv1alpha1.ManageIQ) *corev1.PersistentVolumeClaim {
+func NewPostgresqlPVC(cr *miqv1alpha1.ManageIQ) (*corev1.PersistentVolumeClaim, controllerutil.MutateFn) {
 	labels := map[string]string{
 		"app": cr.Spec.AppName,
 	}
+
 	storageReq, _ := resource.ParseQuantity(cr.Spec.DatabaseVolumeCapacity)
+
+	resources := corev1.ResourceRequirements{
+		Requests: corev1.ResourceList{
+			"storage": storageReq,
+		},
+	}
+
+	accessModes := []corev1.PersistentVolumeAccessMode{
+		"ReadWriteOnce",
+	}
+
 	pvc := &corev1.PersistentVolumeClaim{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "postgresql",
+			Name:      "miq-pgdb-volume",
```

Comment:
Same, why change the name?

Reply:
Also -1 for using miq hardcoded because then it prevents alternate naming.